errors.toml

```diff
@@ -1649,7 +1649,7 @@ workaround = ""
 tags = ["internal", "high"]
 
 [error.DM-sync-unit-36011]
-message = "sharding group for `%s`.`%s` not found"
```

Comment:
why this change ?

Reply:
I think this is caused by our CI didn't find `make terror` is failed 🤔

Reply:
Maybe because I changed `errors_release.txt` and `error_list.go` but forgot to change `errors.toml`😂. https://github.com/pingcap/dm/pull/2086/files#diff-f3483b78d7ad523f4f600b47bcb2ab11f7cfb9e9783344aa73d17c9413c6b5b8R276.

I will change it in https://github.com/pingcap/dm/pull/2106.

Reply:
we can use `make terror_check` cmd to gen errors in `errors_release.txt` and `errors.toml`, no need to write these documents manually 😬